### PR TITLE
Fixed header height issue for grouped headers

### LINKED ISSUE #14424: Grid: column group headers too big
## Description  
If column group headers are used then the height of the headers is too big when no data is loaded.

 * igniteui-angular version: 18.0.2

## Attachments  
https://stackblitz.com/edit/k2ivlx-me1uxb

2024-06-21_12h03_14

image

### PR DESCRIPTION
**Pull request by MarielaTihova**
Resolves #14424 

### Additional information (check all that apply):
 - [x] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [x] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

File: projects/igniteui-angular/src/lib/grids/grid/column-group.spec.ts [hunk 1]
```diff
@@ -65,6 +65,19 @@ describe('IgxGrid - multi-column headers #grid', () => {
             expect(grid.getColumnByName('ContactName').level).toEqual(expectedLevel);
         });
 
+        fit('should initialize a grid with correct header height', () => {
```
Comment (skrustev) [inline projects/igniteui-angular/src/lib/grids/grid/column-group.spec.ts:68]: Leftover `fit` i guess

**Review by skrustev — CHANGES_REQUESTED** [review]

File: projects/igniteui-angular/src/lib/grids/grid/column-group.spec.ts (hunk 1 above)
Comment (MarielaTihova) [reply projects/igniteui-angular/src/lib/grids/grid/column-group.spec.ts]: Resolved with commit [529d4c69d3bfdf923e0e84ad9ae26f91e2156cb1](https://github.com/IgniteUI/igniteui-angular/pull/14621/commits/529d4c69d3bfdf923e0e84ad9ae26f91e2156cb1)

**Comment by dkamburov** [comment]
@MarielaTihova supported versions are 17.2, 18.1 and master. Please create pr for them as well or this is only reproducible in 18.1?

**Comment by MarielaTihova** [comment]
> @MarielaTihova supported versions are 17.2, 18.1 and master. Please create pr for them as well or this is only reproducible in 18.1?

Created a PR for master as discussed: [Fixed header height issue for grouped headers](https://github.com/IgniteUI/igniteui-angular/pull/14788)

**Pull request closed without merging**